pkg/crawl/identify.go

```diff
@@ -30,6 +32,20 @@ import (
 	"go.uber.org/ratelimit"
 )
 
+// vulnerableFindingStrings contains only the Finding values that are considered vulnerable.
+var vulnerableFindingStrings = map[Finding]string{
+	JndiLookupClassName:            "jndiLookupClassName",
```

Comment:
Amazing, yeah, I did it programatically but thought it would be deemed over-engineering so reverted to a hardcoded version. Will make it back to programatic 👌

Reply:
I've switched yours suggestion around, so that the map is the source of truth rather than the if statements in the String method.

This way, when there is a new value that's needed to be added, a maintainer just needs to add a single key and value to the map rather than creating a new if statement in the String method. Defining the map in the codebase also means that we get key uniqueness checking for free.

Let me know if you prefer me to do it the other way and I can do that still.

The new check highlights the fact that we have two different string representations of a finding, lower camel case in Reporter.Report and an upper camel case in the Finding.String method. I'm keen to merge them to all match but don't want to break any workflows by doing so. What would you advise?